public/coffee/ide/history/controllers/HistoryDiffController.coffee

```diff
@@ -9,7 +9,7 @@ define [
 				.restoreDeletedDoc(
 					$scope.history.diff.doc
 				)
-				.success (response) ->
+				.then (response) ->
```

Comment:
I suspect this should be followed by a `{data} = response`, then adjust line 13 to be `... = data.doc_id`